roles/hosted_engine_setup/defaults/main.yml

```diff
@@ -49,6 +49,11 @@ he_force_ip4: false
 he_force_ip6: false
 
 he_pause_host: false
+
+# Pause execution similarly to setting he_pause_host to true, if "add host"
+# failed with status "non_operational". Set to false in your vars file if
+#  you want deploy to fail immediately instead of the default "true".
```

Comment:
I don't think we need to explain it here as we already have the explanation in README

Reply:
ok, done